### DISCUSSION TITLE
Does flux support fooocus?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#3594] [General]
Does flux support fooocus?

**Comment by IPv6** [comment]
Ask Flux developers, they should know! :)

**Comment by mashb1t** [comment]
![ddmcdqs-ecd2b865-fa25-4550-adf0-f5962af5484f~2](https://github.com/user-attachments/assets/a4b969ee-92f2-4509-b855-b0b56f19ae49)

**Comment by Yogesh-DevHub** [comment]
> Does flux support fooocus?

There are multiple forks of fooocus, a few have managed flux having some comfy UI things going in the backend. 

This fork, metercai/SimpleSDXL works with flux but there are few bugs. The developer is active on project and there are updates.
https://youtu.be/IREkiuRpQiY?si=z90l1jqy8wQnW3Xf

I liked this fork, runew0lf/RuinedFooocus flux.1 schnell & flux.1 dev both works, also support lora. I did kind of a detailed video on installation and usage. How much resource it consumed.

Installation and flux schnell video: https://youtu.be/aCT5RLiPpVs?si=k02J875V4IOGFbBa

flux.1 dev video: https://youtu.be/cvcrv21AHO8?si=un3Kt_xQPqj_Qjcf

**Reply by mashb1t** [reply]
thanks @Yogesh-DevHub

@martinenkoEduard keep in mind that runew0lf/RuinedFooocus is quite different compared to this repository. You'll be better off using metercai/SimpleSDXL.

**Reply by DavidDragonsage** [reply]
Yes, I prefer SimpleSDXL because I find Ruined Fooocus to be too divergent from mainline Fooocus. 

It is worth noting that Ruined Fooocus is 834 commits behind mainline Fooocus (I don't know what version of the mainline that translates to). In contrast, SimpleSDXL is completely up-to-date with Fooocus 2.55. I definitely do not want to go back in time and lose the features I have come to rely on in Fooocus 2.55.

Rodney Kleeberg (ILikeToasters) has a great video on installing SimpleSDXL and using it with Flux:
https://www.youtube.com/watch?v=qJxCLIF2rJc
Rodney intends to provide further videos on SimpleSDXL in the future.

At our "Pure Fooocus" Facebook group, we have support for both mainline Fooocus and SimpleSDXL. So far there has been little or no interest in Ruined Fooocus: https://www.facebook.com/groups/fooocus/
